src/index.html

```diff
@@ -11,8 +14,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with a .scss extension cannot be linked directly in HTML. You should compile SCSS to CSS and link the compiled CSS file instead.